dist/components/select/_select.scss

```diff
@@ -45,6 +47,37 @@ $select__triangle-position: $select__h-space;
 
     user-select: none;
 
+
+    // "Down" Triangle
+    // ---
+    //
+    // 1. Absolutely position select element and triangle relative to the parent .c-select
+    // 2. Vertical center the triangle
+
+    &::after {
+        @include triangle(
+            $type: 'triangle',
+            $direction: 'down',
+            $size: $select__triangle-size,
+            $color: $select__border-color
+        );
+
+        content: '';
+
+        position: absolute; // 1, 2
+        top: 0; // 2,
+        right: $select__triangle-position;
+        bottom: 0; // 2
+
+        margin: auto; // 2
+    }
+
+
+    // Select
+    // ---
+    //
+    // 1. Absolutely position select element and triangle relative to the parent .c-select
+
```

Comment:
@jeffkamo What's your thoughts about **Select** component? Does it have similar issue as **Progress-steps** or it looks fine since all children elements have comments and titles?